src/dlgTriggerEditor.cpp

```diff
@@ -27,6 +27,7 @@
 
 #include "dlgActionMainArea.h"
 #include "dlgAliasMainArea.h"
+#include "dlgButtonSelect.h"
```

Comment:
Well if there is a well-defined symbol to test against then use it - but in most cases there is just a "This was introduced in Qt 5.x" reference in the documentation... :smirk: